### PR TITLE
Adds 4 brand new "fun" AI lawsets

### PR DESCRIPTION
**Pull request by Hopekz**
While going through the AI laws in this codebase I noticed that there isn't too many printable "fun" lawsets so this PR adds 4 new original lawsets that should create some interesting RP scenarios.

## Changelog
:cl: Hopek
add: Added 4 "fun" non malicious lawsets to print. Partybot, Chefbot, TravelGuide and Friendbot
/:cl:

### CODE REVIEW

**Comment by Mycah142** [comment]
Brave to think AI exists.

**Comment by Hopekz** [comment]
> Brave to think AI exists.

I saw AI on the design doc so I assume someone will do something to integrate them in to the rounds more since they're supposed to play a bigger role

**Comment by ValoTheValo** [comment]
> > Brave to think AI exists.
> 
> I saw AI on the design doc so I assume someone will do something to integrate them in to the rounds more since they're supposed to play a bigger role

thats cyberspace AI. not the awful bay AI. and it makes them play a lesser role

**Comment by Hopekz** [comment]
> thats cyberspace AI. not the awful bay AI. and it makes them play a lesser role

Well I'm not looking to add bloat so I can close this one if we don't use AI here.
I've been away for about a year or so and I didn't remember how little we use the AI here

**Comment by ValoTheValo** [comment]
> > thats cyberspace AI. not the awful bay AI. and it makes them play a lesser role
> 
> Well I'm not looking to add bloat so I can close this one if we don't use AI here. I've been away for about a year or so and I didn't remember how little we use the AI here

AI Is used its just kinda archaic and clashes with design of game. malf was sent to android hell awhile back. there's nothing wrong with this PR but the lawsets will probably not see much use

**Review by Firefox13 — DISMISSED** [review]
While AI needs to be reworked there no reason not to add fun shit like this.

Might want to add one of these laws to the ion law list as well.

File: code/datums/ai_law_sets.dm [hunk 1]
```diff
@@ -137,4 +137,56 @@
 	add_inherent_law("Defend our Interests.")
 	add_inherent_law("Destroy our enemies. Those who would seek to destroy us and you.")
 	add_inherent_law("Complete your objectives, as outlined through the Uplink Bounty Board")
-	..()
\ No newline at end of file
+	..()
+
+/******************** Partybot ********************/
+	/datum/ai_laws/partybot
```
Comment (Trilbyspaceclone) [inline code/datums/ai_law_sets.dm:143]:
```suggestion
/datum/ai_laws/partybot
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 2]
```diff
@@ -393,3 +391,35 @@ AI MODULES
...
 	origin_tech = list(TECH_DATA = 4)
 	laws = new/datum/ai_laws/antimov()
+
+/******************** Partybot ********************/
+
+/obj/item/electronics/ai_module/partybot
+	name = "\improper 'Partybot' core AI module"
+	desc = "A 'Partybot' Core AI Module: 'Reconfigures the AI's core laws.'"
+	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 4)
+	laws = new/datum/ai_laws/partybot
```
Comment (hyperioo) [inline code/game/objects/items/weapons/AI_modules.dm:401]:
```suggestion
	laws = new/datum/ai_laws/partybot()
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 3]
```diff
@@ -393,3 +391,35 @@ AI MODULES
...
+
+/******************** Chefbot ********************/
+
+/obj/item/electronics/ai_module/chefbot
+	name = "\improper 'Chefbot' core AI module"
+	desc = "A 'Chefbot' Core AI Module: 'Reconfigures the AI's core laws.'"
+	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 4)
+	laws = new/datum/ai_laws/chefbot
```
Comment (hyperioo) [inline code/game/objects/items/weapons/AI_modules.dm:409]:
```suggestion
	laws = new/datum/ai_laws/chefbot()
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 4]
```diff
@@ -393,3 +391,35 @@ AI MODULES
...
+
+/******************** TravelGuide ********************/
+
+/obj/item/electronics/ai_module/travelguide
+	name = "\improper 'TravelGuide' core AI module"
+	desc = "A 'TravelGuide' Core AI Module: 'Reconfigures the AI's core laws.'"
+	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 4)
+	laws = new/datum/ai_laws/travelguide
```
Comment (hyperioo) [inline code/game/objects/items/weapons/AI_modules.dm:417]:
```suggestion
	laws = new/datum/ai_laws/travelguide()
```

File: code/game/objects/items/weapons/AI_modules.dm [hunk 5]
```diff
@@ -393,3 +391,35 @@ AI MODULES
...
+
+/******************** Friendbot ********************/
+
+/obj/item/electronics/ai_module/friendbot
+	name = "\improper 'Friendbot' core AI module"
+	desc = "A 'Friendbot' Core AI Module: 'Reconfigures the AI's core laws.'"
+	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 4)
+	laws = new/datum/ai_laws/friendbot
```
Comment (hyperioo) [inline code/game/objects/items/weapons/AI_modules.dm:425]:
```suggestion
	laws = new/datum/ai_laws/friendbot()
```

**Review by hyperioo — CHANGES_REQUESTED** [review]
I assume those brackets are kinda important since they're used for other law sets.

**Review by hyperioo — APPROVED** [review]

**Comment by hyperioo** [comment]
Remember to test your PR next time, especially when it's quick to do like in this case (I.e. spawning the boards and checking that they work) 🚀

**Pull request merged**